### DISCUSSION TITLE
Components derived from BaseWebComponent cannot consume DateHelper

### DISCUSSION THREAD

**Discussion by ewillman** [microsoft-search/pnp-modern-search#1832] [Q&A]
This comment in the BaseWebComponent implies that components derived from it should be able to consume the DateHelper service.  However, the DateHelper.ServiceKey is only defined within the webpart deployment package, not in the extensibility npm package. In the debugger, I can see that the DateHelper service is registered in the service scope, but have no way to call consume to retrieve a reference to that service:

https://github.com/microsoft-search/pnp-modern-search/blob/b78aba6323a9d12ae2cca39f709f933d0f3f1746/search-extensibility/src/models/layouts/BaseWebComponent.ts#L28

**Comment by wobba** [comment]
@FranckyC any comments?
